modules/capability/module.go

```diff
@@ -27,8 +27,11 @@ import (
 )
 
 var (
+	_ module.AppModule           = (*AppModule)(nil)
 	_ module.AppModuleBasic      = (*AppModuleBasic)(nil)
```

Comment:
I think we can safely remove this assertion and the associated functions

Reply:
because `AppModule` is `AppModuleBasic` :D makes sense!